community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/util.py

```diff
@@ -1683,7 +1687,7 @@ def properties(inst):
             instance_iter = (
                 (inst["name"], properties(inst))
                 for inst in chain.from_iterable(
-                    m["instances"] for m in result.get("items", {}).values()
+                    m["instances"] for m in result.get("items", {}).values() if "instances" in m
```

Comment:
What would be a condition for `"instances" not in m` ?

Reply:
`response.items` is map from `zone` to result. API may return empty/none list of items.
nit. To avoid string literal duplication and improve readability I would recommend to write it as:

`zone.get("instances", []) for zone in result.get("items", {}).values()`

Reply:
Done.